bin/studio-common

```diff
@@ -38,6 +39,17 @@ function document() {
   [ -z "$CI" ] && echo -e "$content" > "/tmp/docs/${name}"
 }
 
+# This is a helper function that allows you to quickly and easily add an alias for
+# a helper function. It will also ensure that a list of aliases are printed when
+# you run `describe`.
+function add_alias() {
+  fun="$1"
+  a="$2"
+
+  eval "alias $a=$fun"
+  [ -z "$CI" ] && echo -e "$a" > "/tmp/aliases/$fun"
```

Comment:
Why not have aliases in CI systems? 🤔

Reply:
This is not _documenting_ the aliases. The alias is still being created on 49.